[run-webkit-tests] Strip allows*HTTPS* logging messages

JonWBedard opened the pull request:
#### 08b7126fcf9f0bba94141fc1afd4b87a628329c5
<pre>
[run-webkit-tests] Strip allows*HTTPS* logging messages
<a href="https://bugs.webkit.org/show_bug.cgi?id=249658">https://bugs.webkit.org/show_bug.cgi?id=249658</a>
rdar://56920527

Reviewed by Alexey Proskuryakov.

* Tools/Scripts/webkitpy/port/darwin.py:
(DarwinPort.stderr_patterns_to_strip): Strip allows*HTTPS* logging messages.
* Tools/Scripts/webkitpy/port/darwin_testcase.py:
* Tools/Scripts/webkitpy/port/ios_simulator.py:
(IOSSimulatorPort.logging_patterns_to_strip): Deleted.
(IOSSimulatorPort.stderr_patterns_to_strip): Deleted.
* Tools/Scripts/webkitpy/port/mac.py:
(MacPort.stderr_patterns_to_strip): Invoke base class&apos;s stderr_patterns_to_strip.
* Tools/Scripts/webkitpy/port/watch_simulator.py:
(WatchSimulatorPort.logging_patterns_to_strip): Deleted.
(WatchSimulatorPort.stderr_patterns_to_strip): Deleted.

Canonical link: <a href="https://commits.webkit.org/258197@main">https://commits.webkit.org/258197@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/1a41b6edfcd77cec66b0e006d7a5fdbe9afa9cd2

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [  ~~🧪 style~~](https://ews-build.webkit.org/#/builders/6/builds/101215 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 ios~~](https://ews-build.webkit.org/#/builders/77/builds/10368 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 mac~~](https://ews-build.webkit.org/#/builders/43/builds/34268 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wpe~~](https://ews-build.webkit.org/#/builders/8/builds/110498 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 🧪 win~~](https://ews-build.webkit.org/#/builders/10/builds/170779 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [  ~~🧪 bindings~~](https://ews-build.webkit.org/#/builders/11/builds/105202 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 ios-sim~~](https://ews-build.webkit.org/#/builders/76/builds/11305 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 mac-AS-debug~~](https://ews-build.webkit.org/#/builders/85/builds/1235 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 gtk~~](https://ews-build.webkit.org/#/builders/36/builds/93619 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/12/builds/108330 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [  ~~🧪 webkitperl~~](https://ews-build.webkit.org/#/builders/19/builds/106998 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/76/builds/11305 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/43/builds/34268 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/93619 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitpy](https://ews-build.webkit.org/#/builders/5/builds/104735 "Passed tests") | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/76/builds/11305 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/43/builds/34268 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/36/builds/93619 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/81/builds/4009 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/43/builds/34268 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/82/builds/4059 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/85/builds/1235 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/80/builds/10160 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/43/builds/34268 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/79/builds/5813 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/2955 "Built successfully and passed tests") | | | | 
<!--EWS-Status-Bubble-End-->

aproskuryakov commented on Tools/Scripts/webkitpy/port/watch_simulator.py:
```diff
@@ -98,6 +98,3 @@ def developer_dir(self):
         return []
-
-    def stderr_patterns_to_strip(self):
-        return []
```
It feels inconsistent to have stdout here, but not stderr. Should subclasses call through to super class explicitly, so that platform specific patterns could be easily added?

JonWBedard replied:
I think so, I'll remove these empty returns everywhere

JonWBedard committed:
[run-webkit-tests] Strip allows*HTTPS* logging messages
https://bugs.webkit.org/show_bug.cgi?id=249658
rdar://56920527

Reviewed by Alexey Proskuryakov.

* Tools/Scripts/webkitpy/port/darwin.py:
(DarwinPort.stderr_patterns_to_strip): Strip allows*HTTPS* logging messages.
* Tools/Scripts/webkitpy/port/darwin_testcase.py:
* Tools/Scripts/webkitpy/port/ios_simulator.py:
(IOSSimulatorPort.logging_patterns_to_strip): Deleted.
(IOSSimulatorPort.stderr_patterns_to_strip): Deleted.
* Tools/Scripts/webkitpy/port/mac.py:
(MacPort.stderr_patterns_to_strip): Invoke base class's stderr_patterns_to_strip.
* Tools/Scripts/webkitpy/port/watch_simulator.py:
(WatchSimulatorPort.logging_patterns_to_strip): Deleted.
(WatchSimulatorPort.stderr_patterns_to_strip): Deleted.

Canonical link: https://commits.webkit.org/258197@main
